core/templates/services/translation-file-hash-loader.service.ts

```diff
@@ -32,10 +33,10 @@ angular.module('oppia').factory('TranslationFileHashLoaderService', [
         options.key,
         options.suffix
       ].join('');
-      return $http.get(
+      return TranslationsBackendApiService.loadTranslationFileHash(
         UrlInterpolationService.getStaticAssetUrl(fileUrl)
       ).then(function(result) {
```

Comment:
```suggestion
      return TranslationsBackendApiService.loadTranslationFileHash(fileUrl).then(function(result) {
```

(Let's only have the filUrl as a param here and create URL in the main function.)

Reply:
Done!